### ISSUE TITLE
Stop installation if it's running by root/sudo

### ISSUE THREAD

**Issue by Defman21** [Komodo/KomodoEdit#521]
It's just **too dangerous**. The installation process should stop itself if it's running by root/sudo or at least give a warn message. Related to http://forum.komodoide.com/t/your-komodoedit-profile-cannot-be-loaded-missing-or-inaccessible/1904

**Comment by Naatan** [comment]
I don't agree we should stop running altogether, but we ought to give a warning.

**Comment by Defman21** [comment]
What reasons are there to don't prevent installing by root? You're able to install it to `/opt/` as normal user and then use sudo to run it as root user or don't use sudo to run it as normal user.

**Comment by johngilbrough** [comment]
I've wasted more than two days so far fighting the Komodo 9.3 permissions bug on Ubuntu 15.10. The last attempt was to purge everything and then re-install 9.3 into my home directory (instead of where it belongs in the /opt directory) - and I'm still getting the goddamn "Error: Access was denied while trying to open files in your profile directory." 

if I rm -r .komodo followed by Komodo-Edit-9/bin/komodo, the same error occurs.  The six files left in .komodo/ all have my permissions.  So what "profile directory" is komodo getting denied from???  It's not .activestate - those are my permissions.  And why would someone write something so stupid as to not tell you where the problem is?

The errors that occur in the terminal window are "(process:5698): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed
Error: Access was denied while trying to open files in your profile directory."

And there's another error that shows up in the middle of nowhere without a window frame that is very hard to read (but it has a button):

![image](https://cloud.githubusercontent.com/assets/226320/11845093/1e47bf1a-a3e0-11e5-8862-018ab6535a7f.png)

Would you be kind enough to send me an email when this get solved?  Or if not, can you recommend another editor that works?  - Thank you!

**Comment by Defman21** [comment]
It's `~/.komodoedit`.

**Comment by Naatan** [comment]
@johngilbrough try the following

In your terminal, type the following:

`ls -ahl ~`

This should list all files in your home directory, and it should tell you what user and group owns each file. Make note of them.

Now execute the following, using the information gathered in the previous step:

`sudo chown -R myusername.myusergroup ~/.komodoedit`

replacing "myusername" and "myusergroup" with your username and usergroup. Note there is a dot in between.

That should solve any permission issues you are having.

**Comment by Naatan** [comment]
Renamed the bug; to be clear:

 * Komodo can be installed using sudo (I do this myself)
 * Komodo can be ran as root, root is just another user. We should probably warn users if they do this.
 * Komodo should NOT be run under sudo, this is what creates problems, we should outright prevent this from happening.
